lib/infobox/node.js

```diff
@@ -125,15 +125,16 @@ define(['sorttable', 'snabbdom', 'd3-interpolate', 'moment', 'helper', 'utils/no
           var field = d[row.value];
           if (nodef['show' + row.value] !== undefined) {
             field = nodef['show' + row.value](d);
+          } else if (typeof row.value === 'function') {
```

Comment:
Wäre es nicht besser, zunächst auf function zu überprüfen?
ansonsten werden die object-key request ziemlich lang:
```
nodef["show"+console.log]
```
Entspricht:
```
nodef["showfunction log() {
    [native code]
}"]
```

Reply:
i will change it. I don't think it has a impact on performance, but the logic looks better.

Reply:
war mal so frei ... kannst mit dem commit machen was du willst ...